providers/aws/services/autoscaling/autoscaling_find_secrets_ec2_launch_configuration/autoscaling_find_secrets_ec2_launch_configuration.py

```diff
@@ -0,0 +1,46 @@
+import os
+from base64 import b64decode
+
+from detect_secrets import SecretsCollection
+from detect_secrets.settings import default_settings
+
+from lib.check.models import Check, Check_Report
+from providers.aws.services.autoscaling.autoscaling_client import autoscaling_client
+
+
+class autoscaling_find_secrets_ec2_launch_configuration(Check):
+    def execute(self):
+        findings = []
+        for configuration in autoscaling_client.launch_configurations:
+            report = Check_Report(self.metadata)
+            report.region = configuration.region
+            report.resource_id = configuration.name
+            report.resource_arn = configuration.arn
+
+            if configuration.user_data:
+                temp_user_data = "/tmp/user_data.txt"
+                user_data = b64decode(configuration.user_data).decode("utf-8")
+
+                with open(temp_user_data, "w") as f:
+                    f.write(user_data)
+                    f.close()
+
+                secrets = SecretsCollection()
+                with default_settings():
```

Comment:
Review the secrets detector base configuration for Prowler 2

Reply:
Check it here https://github.com/prowler-cloud/prowler/blob/master/include/secrets_detector#L27-L50

Reply:
Good catch, but the only filter is the hex entropy that is the same as the default one:

> --hex-limit [HEX_LIMIT]
                        Sets the entropy limit for high entropy strings. Value
                        must be between 0.0 and 8.0, defaults to 3.0.

Reply:
Also the following but we use the default value 👌 
```
  --base64-limit [BASE64_LIMIT]
                        Sets the entropy limit for high entropy strings. Value
                        must be between 0.0 and 8.0, defaults to 4.5.
```